### ISSUE TITLE
GCAM-USA fails to run in GCAM

### ISSUE THREAD

**Issue by pralitp** [JGCRI/gcamdata#977]
This was a known issue that we chose to forgo on the assumption that we are overhauling the GCAM-USA anyhow.  However it seems that overhaul may miss the GCAM release so we will need to go back and fix it after all to have a working version in the next release.

**Comment by kvcalvin** [comment]
@pralitp: this one was fixed, right?

**Comment by pralitp** [comment]
Yup. In the gcam repo though. I was waiting to close the issue until I back port to github but if you want to just knock it out now,  go for it.
